### PR TITLE
add rate limiting guide

### PR DESCRIPTION
**Pull request by povilasv**
This PR adds rate limiting guide

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

Closes https://github.com/kubeshop/kusk-gateway/issues/423

### CODE REVIEW

**Review by jasmingacic — APPROVED** [review]
This is looking good.

@aabedraba can you pass it further for review before we merge it

File: docs/guides/rate-limit.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    get:
+      operationId: getHello
+      x-kusk:
+        rate_limit:
+          requests_per_unit: 2
+          unit: minute
+      ..
+```
+
+See all available Rate Limiting configuration options in the [Extension Reference](../../reference/extension/#rate-limiting).
```
[the remark this answers is not in the document]
Comment (povilasv) [reply docs/guides/rate-limit.md:36]: It does resolve in preview for me https://deploy-preview-459--kusk-docs-preview.netlify.app/reference/extension/#rate-limiting

**Review by kylehodgetts — APPROVED** [review]

File: docs/guides/rate-limit.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kylehodgetts) [reply docs/guides/rate-limit.md:36]: the link is correct

[the remark this answers is not in the document]
Comment (jasmingacic) [reply docs/guides/rate-limit.md:36]: the link is working
don't worry about it

**Review by jfermi — APPROVED** [review]

**Pull request merged**